### PR TITLE
build/configs/rtl8730e: add Linux download support

### PR DESCRIPTION
**Pull request by SandyYeow**

### CODE REVIEW

File: build/configs/rtl8730e/hello/defconfig [hunk 1]
```diff
@@ -281,9 +281,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="48,76,4,3260,3260,512,1024,8,"
-CONFIG_FLASH_PART_TYPE="none,none,none,kernel,kernel,smartfs,ftl,bootparam,"
-CONFIG_FLASH_PART_NAME="bl1,reserved,system_data,kernel,kernel,userfs,reserved,bootparam,"
+CONFIG_FLASH_PART_SIZE="128,1792,4728,512,1024,8,"
```
Comment (sunghan-chang) [inline build/configs/rtl8730e/hello/defconfig:284]: Let's use the same size for both  kernels.

File: build/configs/rtl8730e/.flashSpec.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<flash name="ameba flash" virtual="false">
+    <!-- path attribute value must be relative path from project root -->
+    <executors>
+        <execute name="flash" path="${PROJECT_PATH}/os" command="make download" os="linux" bit="all"></execute>
+    </executors>
+    <options>
+        <option desc="Flash all binaries and images">ALL</option>
+        <option desc="Flash TizenRT km4 bootloader">BL1</option>
+        <option desc="Flash TizenRT kernel">KERNEL</option>
+        <option desc="Flash TizenRT ota">OTA</option>
```
Comment (sunghan-chang) [inline build/configs/rtl8730e/.flashSpec.xml:26]: There is no OTA partition and no option for this in the download script. Could you check whether this is necessary or not?

**Review by sunghan-chang — COMMENTED** [review]
The title says `1st release`. Do you have next release plan? Could you let me know what 2nd release will include?

**Comment by SandyYeow** [comment]
> The title says `1st release`. Do you have next release plan? Could you let me know what 2nd release will include?

**Comment by SandyYeow** [comment]
> The title says `1st release`. Do you have next release plan? Could you let me know what 2nd release will include?

Dear Mr. Chang, 
Currently there is no plan for 2nd release. Unless there is further request, then we will have 2nd release. Thank you.

**Comment by sunghan-chang** [comment]
> Dear Mr. Chang,
Currently there is no plan for 2nd release. Unless there is further request, then we will have 2nd release. Thank you.

Then, I suggest `build/configs/rtl8730e: add Linux download support` for a title.

File: build/configs/rtl8730e/.flashSpec.xml (hunk 2 above)
Comment (SandyYeow) [reply build/configs/rtl8730e/.flashSpec.xml]: Dear Mr. Chang,

Noted. The changes have submitted.
Thanks.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**